### PR TITLE
Abandoned Interdyne research base

### PR DESCRIPTION
**Pull request by DaydreamIQ**
## About The Pull Request
This PR adds the Interdyne Spinward Research Base! An abandoned facility left adrift after the outbreak of a deadly virus. EOB made mention about Romerol's origins and given how little its actually referenced I thought I'd make my first ever mapping PR to add just a tiny bit more to one of my favourite Syndie factions. Casual explorers won't leave empty handed either so there's a little something for everyone

Shoutout to the mapping event channel guys for helping with all this

### Mapping March
Ckey to receive rewards: DaydreamIQ

## Why It's Good For The Game
More lore ruins are good, as are fun little places you can feasibly choose to live in if that suits your fancy.
## Changelog
:cl:
add: Added the Abandoned Interdyne facility, explore at your own peril

/:cl:

### CODE REVIEW

**Comment by Jacquerel** [comment]
Could you upload a screenshot with areas disabled? Or some in-game shots? 
It makes it a little hard to parse.

**Comment by DaydreamIQ** [comment]
My bad, how's that?

**Comment by san7890** [comment]
the areas screencap is concerning, do you have area passthroughs as well as the turf passthroughs?

also, either center the map to have a 1 tile margin of "space" around it or shrink the map size entirely, having extra padding on only two ends feels weird

image

**Comment by Elli-Skala** [comment]
SmartSelect_20230326_014011_Chrome

If you moved theses two containers down one tile it would be the perfect place to hide ~~a bee plushie~~  cool loot behind them

**Comment by DaydreamIQ** [comment]
> If you moved theses two containers down one tile it would be the perfect place to hide ~a bee plushie~ cool loot behind them

image
:^)

**Review by spockye — COMMENTED** [review]
hello! I went through the ruin and found a couple errors

image
stacked APC's. you also don't need to give them terminals! they spawn in by themselves for APC's. they do not however for SMES units. 
a quick thing about apc's. they work for the entire area that they are placed inside. Having multiple apc's inside the same area will quickly bug them out. This entire ruin is one area and has 15 apc's in total. you should probably either give each room with an apc its own area or remove all but one apc. 

image
you should always put empty plating under windows! if yer not using a spawner, remember to add grilles as well!

image
This couch is completely blocked in, making the only way to get to it, climbing over the table, or unwrenching the machines on either side. 

image
Both chem masters are blocked in, making them unusable! There's also a wall missing in the corner. Misplaced barricade next to the entrance?  theres also no need for all those cables everywhere! 

image
when putting down posters, or any wallmount that don't have premade directions, remember to pixelshift them! they should generally be at +- 32. Theres also three identical posters in that room

image
ID's should generally be pretty destinct, so they don't accidentally use the same as something else. "interdyne_vault" might work better!

image
Couple missing cables. Theres also a lot of cables in the walls, and cables that don't serve any purpouse. try cutting back on them!

image
it's nice to have atmos, but remember that they need a place to get the air from, and to dump the scrubbed gases. else they don't really do much. 

I would recommend replacing the corner walls with non-diagonal walls, so they don't turn in-game. it can look good, but generally doesn't.

(lights also don't need to be connected to the cable network)

**Comment by DaydreamIQ** [comment]
> hello! I went through the ruin and found a couple errors

Thanks for looking through that, I'm working on fixing those issues now although I'm already not fantastic with atmos, you think it would be better to clear out the atmos piping out if I can't put in an airtank for it all?

File: code/game/area/areas/ruins/space.dm [hunk 1]
```diff
@@ -581,6 +586,8 @@
 /area/ruin/space/has_grav/travelers_rest
 	name = "\improper Traveler's Rest"
 
+
+
```
Comment (Maurukas) [inline code/game/area/areas/ruins/space.dm:590]: extra lines, probably more conflict leftovers.

File: code/datums/ruins/space.dm [hunk 2]
```diff
@@ -401,6 +401,12 @@
 	name = "Pod Crash"
 	description = "A tragic display of what happens to drivers who pda and pod."
 
+/datum/map_template/ruin/space/Interdyne
```
Comment (Maurukas) [inline code/datums/ruins/space.dm:404]:
```suggestion
/datum/map_template/ruin/space/interdyne
```

Caps

**Review by Maurukas — CHANGES_REQUESTED** [review]
<details>
<summary>You have a loop in your grid here:</summary>

The input and output side of the SMES are connected, this causes a loop that can starve all the APCs in a powernet very quickly.

image

File: config/spaceruinblacklist.txt
[the remark this answers is not in the document]
Comment (DaydreamIQ) [reply config/spaceruinblacklist.txt:1]: Oh lord I messed that up royally, guess that's what I get for not using the resolve editor properly

**Comment by DaydreamIQ** [comment]
> You have a loop in your grid here:
> The input and output side of the SMES are connected, this causes a loop that can starve all the APCs in a powernet very quickly.
> 
Would this better?
24eeefcf3fe98589b79b69f32a349ccb

File: config/spaceruinblacklist.txt [hunk 3]
```diff
@@ -66,13 +68,3 @@
...
-#_maps/RandomRuins/SpaceRuins/forgottenship.dmm
-#_maps/RandomRuins/SpaceRuins/hellfactory.dmm
-#_maps/RandomRuins/SpaceRuins/space_billboard.dmm
-#_maps/RandomRuins/SpaceRuins/spinwardsmoothies.dmm
-#_maps/RandomRuins/SpaceRuins/dangerous_research.dmm
-#_maps/RandomRuins/SpaceRuins/the_faceoff.dmm
-#_maps/RandomRuins/SpaceRuins/atmosasteroidruin.dmm
-#_maps/RandomRuins/SpaceRuins/prey_pod.dmm
-#_maps/RandomRuins/SpaceRuins/travelers_rest.dmm
-#_maps/RandomRuins/SpaceRuins/whiteshipruin_box.dmm
```
Comment (Maurukas) [inline config/spaceruinblacklist.txt:78]: Something's gone wrong again here, these should not be deleted.

Comment (DaydreamIQ) [reply config/spaceruinblacklist.txt:78]: The duplicate ones? I only removed stuff that appeared twice and shifted the rest in alphabetically. Unless that's intended in which case FUCK

**Review by Maurukas — APPROVED** [review]
